projects/rero/ng-core/src/lib/record/search/record-search.component.html

```diff
@@ -31,7 +31,7 @@
       </a>
     </li>
   </ul>
-  <div *ngIf="!isLoading; else loading">
+  <div *ngIf="!isLoading; else loading" class="container ">
```

Comment:
Same here, *.container* must not be here.

Reply:
Done following your advices